cha/sota/test/procedure.tex

```diff
@@ -1,36 +1,72 @@
-\subsection{Procedure}\label{subsec:procedure}
+\section{Procedure}\label{subsec:procedure}
 
-Referring back to \jjwarning{Ref to the list from the whywetest.tex}, the
-first aspect we wanted to test was offset between playing devices.
+\subsection{Test sample}
+We use the song ``Goodness Gracious'' by ``Ellie Goulding''.  We didn't
+want the apps to change song, possibly causing a resync, while testing.
+To avoid that possibility we loop the song for 30 minutes within
+a single mp3 file.
+
+\subsection{Connecting}
+Since the apps being tested are network connected systems, we have to
+connect them before the test can begin. The connection procedure is
+slightly different for each app. Common for both apps is that we use
+3 devices to test, one is the ``master'' while the others are
+``speakers''. The ``speakers'' are the ones connected to the computer,
```

Comment:
Bare for at være eksplicit, så skriv at de er connected med det kabel vi beskrev tidligere.